cylc/flow/task_pool.py

```diff
@@ -2043,6 +2144,23 @@ def force_trigger_tasks(
             self.add_to_pool(itask)
             self._force_trigger(itask)
 
+    def spawn_parentless_sequential_xtriggers(self):
+        """Spawn successor(s) of parentless wall clock satisfied tasks."""
+        while self.xtrigger_mgr.sequential_spawn_next:
+            taskid = self.xtrigger_mgr.sequential_spawn_next.pop()
+            self.xtrigger_mgr.sequential_has_spawned_next.add(taskid)
+            itask = (
+                self._get_task_by_id(taskid)
+                or self._get_task_by_id(taskid)
+            )
```

Comment:
```suggestion
            itask = self._get_task_by_id(taskid)
```

Reply:
Ah, silly me.. fixed and amended associated commit